0xProject/0x-monorepo
# Add devdoc to all contract artifacts

Resolves #1906.  Resolves #1899.

## Description

* Add devdocs to all contract-artifacts
* Update contract-artifacts scripts to require devdoc
* Does NOT introduce bytecode changes. I noticed that some of the `*.sol` files in `development` branch generate different bytecode from artifacts in `development` branch, and manually removed the changes.
* Fixes a small bug in the contract-artifacts `copy` script where it was outputting to the wrong path

@feuGeneA I cherry-picked your commit from #1878 to hopefully minimise merge conflicts!

## Testing instructions

* Ensure no bytecode changes
* Ensure all artifacts have devdocs
* Run unit tests

## Types of changes





 * New feature (non-breaking change which adds functionality)



## Checklist:
